### ISSUE TITLE
error in web page when empty yaml 

### ISSUE THREAD

**Issue by grahamdyson** [DevSnicket/eunice#151]
When initial or edited yaml is empty, the interactive web page will raise an error instead of showing an empty editor and visualisation.

```
"Cannot destructure property 'dependsUpon' of 'undefined' as it is undefined.

TypeError: Cannot destructure property 'dependsUpon' of 'undefined' as it is undefined.
    at createItemFromObject (webpack-internal:///./node_modules/@devsnicket/eunice-dependency-and-structure/createStackFromYaml/createStackStructureFromYaml.js:101:2)
    at createLevelFromIdentifierOrItem (webpack-internal:///./node_modules/@devsnicket/eunice-dependency-and-structure/createStackFromYaml/createStackStructureFromYaml.js:74:15)
    at createStackFromParsedYaml (webpack-internal:///./node_modules/@devsnicket/eunice-dependency-and-structure/createStackFromYaml/createStackStructureFromYaml.js:14:5)
    at __webpack_exports__.default (webpack-internal:///./node_modules/@devsnicket/eunice-dependency-and-structure/createStackFromYaml/index.js:15:93)
    at _default (webpack-internal:///./parseStackFromYaml.js:14:64)
    at _default (webpack-internal:///./parseStackWithDependencyDirectionAndMutualStackFromYaml.js:16:50)
    at setStateFromValue (webpack-internal:///./createYamlInputElement.js:20:88)
    at eval (webpack-internal:///./node_modules/@devsnicket/eunice-test-harnesses/codeEditor/createEditorForLanguage.js:108:10)
    at Emitter.fire (webpack-internal:///./node_modules/monaco-editor/esm/vs/base/common/event.js:497:34)
    at eval (webpack-internal:///./node_modules/monaco-editor/esm/vs/editor/browser/widget/codeEditorWidget.js:987:110)"
```

**Comment by grahamdyson** [comment]
released in [JavaScript 7.11.1](https://devsnicket.com/eunice/javascript/releases#7.11.1)

**Comment by grahamdyson** [comment]
released in [C# 1.16.2](https://devsnicket.com/eunice/csharp/releases#1.16.2)
